packages/atomic/src/components/commerce/facets/e2e/page-object.ts

```diff
@@ -2,7 +2,10 @@ import type {Page} from '@playwright/test';
 import {BasePageObject} from '../../../../../playwright-utils/base-page-object';
 
 export class AnyFacetPageObject<
-  FacetType extends 'atomic-commerce-category-facet' | 'atomic-commerce-facet',
+  FacetType extends
+    | 'atomic-commerce-category-facet'
+    | 'atomic-commerce-facet'
+    | 'atomic-commerce-numeric-facet',
```

Comment:
Should we also add `atomic-commerce-timeframe-facet` ?

Reply:
Adding them only on demand when adding the test suite for a specific facet component